libobs/obs-source.c

```diff
@@ -155,8 +155,17 @@ obs_module_t *obs_source_get_module(const char *id)
 	return NULL;
 }
 
+extern const struct obs_source_info group_info;
+extern const struct obs_source_info scene_info;
```

Comment:
The usage of `extern` should be reduced not increased.

Maybe put the ids as macro in `obs-internal.h` to avoid this.